filebeat/tests/system/test_modules.py

```diff
@@ -67,6 +67,14 @@ def init(self):
 
         self.index_name = "test-filebeat-modules"
 
+        body = {
+            "transient": {
+                "script.max_compilations_rate": "100/1m"
```

Comment:
CI is failing with:

```
Too many dynamic script compilations within, max: [100/1m]
```

So looks like even `100/1m` is not enough 😬😬😬

Reply:
I set it now to 1000/1m and at least locally it works. This is mainly to get test back to green. We should figure out about the why afterwards.